Makefile

```diff
@@ -223,6 +228,17 @@ $(BUILD_DIR)/public/js/newtab.js: $(SOURCE_FILES)
 shared/data/bundled/tracker-lookup.json:
 	node scripts/bundleTrackers.mjs
 
+# Rebuild content-scope-scripts if it's a local checkout (.git is present), but
+# not otherwise. That is important, since content-scope-scripts releases often
+# have newer source files than build files.
+CONTENT_SCOPE_SCRIPTS_SOURCE =
+ifneq ("$(wildcard node_modules/@duckduckgo/content-scope-scripts/.git/)","")
+	CONTENT_SCOPE_SCRIPTS_SOURCE += $(shell find node_modules/@duckduckgo/content-scope-scripts/src/ node_modules/@duckduckgo/content-scope-scripts/inject/ node_modules/@duckduckgo/content-scope-scripts/package.json -type f -not -name "*~")
+endif
+
+node_modules/@duckduckgo/content-scope-scripts/build/$(browser)/inject.js: $(CONTENT_SCOPE_SCRIPTS_SOURCE)
+	cd node_modules/@duckduckgo/content-scope-scripts; npm install; npm run build
```

Comment:
Shouldn't we split out the install and build steps here?

Reply:
No, I think this is good enough. Also, Make goes back to the PWD after each line, so we'd have to prepend each line with the `cd node_modules/@duckduckgo/content-scope-scripts;`.

Reply:
I mean that `npm ci` should be a separate target, so we don't have to re-run `npm install` on every file change in that dir.

Reply:
Ah right, good point. Done.